release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -3226,3 +3226,51 @@ Issued: 2020-07-21
 An update for openshift is now available for {product-title} 4.4. Details of the
 update are documented in the
 link:https://access.redhat.com/errata/RHSA-2020:2927[RHSA-2020:2927] advisory.
+
+[[ocp-4-4-14]]
+=== RHBA-2020:3075 - {product-title} 4.4.14 Bug Fix Update
+
+Issued: 2020-07-28
+
+{product-title} release 4.4.14 is now available. The list of container images and
+bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:3075[RHBA-2020:3075] advisory.
+The RPM packages included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:3074[RHBA-2020:3074] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5251061[{product-title} 4.4.14 container image list]
+
+[[ocp-4-4-14-upgrading]]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.4 cluster to this latest release, see
+xref:../updating/updating-cluster-between-minor.html#update-upgrading-web_updating-cluster-between-minor[Updating a cluster by using the web console]
+for instructions.
+
+[IMPORTANT]
+====
+If you are upgrading to this release from {product-title} 4.3.3 or earlier, you
+must restart all Pods after the upgrade is complete.
+
+This is because the service CA is automatically rotated as of {product-title}
+4.3.5. The service CA is rotated during the upgrade, and a restart is required
+afterward to ensure that all services use the new service CA before the
+previous service CA expires.
+
+After this one-time manual restart, subsequent upgrades and rotations will
+ensure restart before the service CA expires without requiring manual
+intervention.
+====
+
+[[RHSA-2020-3078]]
+=== RHSA-2020:3078 - Low: {product-title} 4.4 Security Update
+
+Issued: 2020-07-28
+
+An update for ose-cluster-machine-approver-container is now available for
```

Comment:
`ose-cluster-machine-approver-container`

Reply:
Hm, doesn't seem to be the convention in the RNs (though I do thinks there's merit to including backticks). I'll ping Cody on this subject but leave it alone for now :) Thank you!